### PR TITLE
Convert spacefinder debug tools to TS

### PR DESCRIPTION
**Pull request by Jakeii**
## What does this change?
Convert spacefinder debug tools to TS

## Does this change need to be reproduced in dotcom-rendering ?

- [x] No
- [ ] Yes (please indicate your plans for DCR Implementation)

## What is the value of this and can you measure success?
Less JS more TS

### Does this affect other platforms?

- [ ] AMP 
- [ ] Apps
- [ ] Other (please specify)

### Does this affect GLabs Paid Content Pages? Should it have support for Paid Content?

- [ ] No
- [ ] Yes (please give details)

### Does this change break ad-free?

- [ ] No
- [ ] It did, but tests caught it and I fixed it
- [ ] It did, but there was no test coverage so I added that then fixed it

### Does this change update the version of CAPI we're using?

- [ ] No, all the existing database files are just fine
- [ ] Yes, and I have [re-run all the tests locally and checked in all the updated data/database/xyz files](https://github.com/guardian/frontend/blob/main/docs/03-dev-howtos/15-updating-test-database.md)

### Accessibility test checklist

- [ ] [Tested with screen reader](https://accessibility.gutools.co.uk/testing/web/screen-readers/)
- [ ] [Navigable with keyboard](https://accessibility.gutools.co.uk/testing/web/keyboard-navigation/)
- [ ] [Colour contrast passed](https://accessibility.gutools.co.uk/testing/web/colour-contrast/)

### Tested

- [x] Locally
- [ ] On CODE (optional)

### CODE REVIEW

**Comment by mxdvl** [comment]
Cool!

File: .github/workflows/commercial-e2e.yml [hunk 1]
```diff
@@ -4,6 +4,7 @@ on:
         paths:
             - static/src/javascripts/projects/common/modules/commercial/**
             - static/src/javascripts/projects/commercial/**
+            - static/src/javascripts/projects/common/modules/spacefinder-*
```
Comment (zekehuntergreen) [inline .github/workflows/commercial-e2e.yml:7]: Is the idea to add some tests for debug mode?

**Review by zekehuntergreen — APPROVED** [review]
Looks good! 🎉

**Review by arelra — APPROVED** [review]
📈

File: .github/workflows/commercial-e2e.yml (hunk 1 above)
Comment (Jakeii) [reply .github/workflows/commercial-e2e.yml:7]: I thought it best to run tests if spacefindery stuff changes

**Pull request merged**